modules/ROOT/pages/asset-types.adoc

```diff
@@ -6,23 +6,54 @@ endif::[]
 
 The following asset types are supported in Exchange:
 
-* Connectors: Packaged connectivity to an Anypoint Platform endpoint using third-party APIs and standard integration protocols
+* API Groups
++
+A set of APIs bundled into a single asset.
+* API Spec Fragments
++
+A RAML document that has a version and an identifier but is not in itself a complete RAML specification. API spec fragments are also known as _RAML fragments_. Exchange supports JSON, RAML, or OpenAPI Specification (OAS) files.
```

Comment:
IMHO: I would remove the word RAML from the first sentence, it should be more agnostic, it could be something like:

`A piece of a specification that can be used to build a bigger specification, taking advantage of the re-usability`

Reply:
+1

Reply:
How about:
A part of a specification that can be used to build a larger specification while taking advantage of reusability.